### PR TITLE
Update to mariadb

### PR DESCRIPTION
**Pull request by madhulika95b**
@ldko @somexpert 

This PR upgrades the db from 'mysql' to 'mariab'

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,7 +11,8 @@ ENV PYTHONPATH /app:/app/coda
 RUN mkdir /app
 WORKDIR /app
 
-RUN apt-get update -qq && apt-get install -y mysql-client netcat
+RUN apt-get update -qq && \
+    apt-get install -y  mariadb-server mariadb-client libmariadbclient-dev libssl-dev netcat
```
Comment (somexpert) [inline Dockerfile:15]: So did you try without changing this line? I don't think we needed to change the client, mysql-client should work just fine connecting to mariadb.

Comment (madhulika95b) [reply Dockerfile:15]: you are right! `mysql-client` works just fine. I will revert it to what it was.

**Review by somexpert — APPROVED** [review]
Looks good to me! :thumbsup:

**Review by ldko — APPROVED** [review]
works for me! :+1:  Thank you, Madhulika!

**Pull request merged**